hack/convert-output-to-tsv.py

```diff
@@ -0,0 +1,42 @@
+# A script to convert transformed data into TSV files
+# Note: Before uploading to Terra, manually go in and update PK column name to be prefixed by "entity:"
+
+import json
+import csv
+from os import listdir
+import sys
+
+if (len(sys.argv) < 3):
+    print "Please provide the input directory and output directory as arguments!"
+
+table_names = ["hles_dog", "hles_owner"]
```

Comment:
Does this need to be hard-coded, or could we just `ls` the output / tables directory?

Reply:
Good point. I'll have it check the directory 👍